### PR TITLE
Updated Dockerfile for ports

### PR DESCRIPTION
**Pull request by ashish123mahajan**
Declared default ports and removed the health check URL which is on 8080, as we are specifying the port and health check on runtime.

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -17,13 +17,12 @@ FROM openjdk:8 as final
...
 WORKDIR /app
 
-ENV HEALTHCHECKURL http://localhost:8080/actuator/health
 ARG JAVA_OPTS
 
 COPY ./a2d2-api/target/a2d2-api.war /app/a2d2-api.war
 COPY --from=maven /usr/src/client_repo /root/.m2/repository
 COPY --from=maven /usr/src/services /app/services
 
-HEALTHCHECK --interval=60s --timeout=30s --start-period=30s --retries=3 CMD curl -f $HEALTHCHECKURL 2>&1 | grep UP || exit 1
```
Comment (hankwallace) [inline Dockerfile:27]: Why are you removing the health check? These were added to ensure that docker could determine if the service is healthy.

**Review by marianbuenosayres — APPROVED** [review]

**Review by hankwallace — CHANGES_REQUESTED** [review]
Do not merge this until we discuss/resolve the removal of the health check.

File: Dockerfile (hunk 1 above)
Comment (ashish123mahajan) [reply Dockerfile:27]: Hank, removed the health check URL which is on 8080, as we are specifying the port and health check on runtime.

Comment (hankwallace) [reply Dockerfile:27]: That doesn't explain why you are removing it. It's defined on port 8080, which is one of the ports allowed for this container. The team went through all services and added these health checks in the past.

Comment (ashish123mahajan) [reply Dockerfile:27]: I have reverted changes for healthcheck, Please review it now.

**Review by hankwallace — APPROVED** [review]

**Pull request merged**